conda-forge.yml

```
@@ -1,13 +1,18 @@
+bot:
+  automerge: true
 build_platform:
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
   osx_arm64: osx_64
+conda_build:
+  pkg_format: '2'
 conda_forge_output_validation: true
-test_on_native_only: true
-os_version:
-  linux_64: cos7
 github:
   branch_name: main
   tooling_branch_name: main
-conda_build:
-  pkg_format: '2'
-bot:
-  automerge: true
+os_version:
+  linux_64: cos7
+provider:
+  linux_aarch64: default
+  linux_ppc64le: default
+test_on_native_only: true
```

Proposing replacing [deprecated key]( https://github.com/conda-forge/conda-smithy/blob/cc803a4088efc4c7e7971e908be6b693c9894276/CHANGELOG.rst#L911-L914 )

```suggestion
test: native_and_emulated
```